### ISSUE TITLE
 IC-Light tab not available for me

### ISSUE THREAD

**Issue by reborn15** [QodiCat/IC-Light-sd-webui#9]
I Install all model and check name, but  IC-Light tab not available for me
![lc 2](https://github.com/QodiCat/IC-Light-SD-WebUI/assets/172972501/5c031159-8e17-4172-bec1-a25c7178c235)
![lc 1](https://github.com/QodiCat/IC-Light-SD-WebUI/assets/172972501/cdeb36f2-cb8f-4edd-9f0f-0f53be9ef51c)

**Comment by pttrwt** [comment]
Looking at console that will show error, maybe like this
![ภาพ](https://github.com/QodiCat/IC-Light-SD-WebUI/assets/40859919/464bb097-5fc1-46a7-9bf9-f90050a1941e)

**Comment by pttrwt** [comment]
Looking at console that will show error, maybe like this.
![ภาพ](https://github.com/QodiCat/IC-Light-SD-WebUI/assets/40859919/e9dcb1bc-22d0-4da0-a78e-6e6b06c54f84)

**Comment by AZoches** [comment]
did you found any solvation

**Comment by vasiluz** [comment]
TLDR for Win:
```
venv\Scripts\activate.bat
pip install diffusers==0.27.2
```

I had the same issue.  Solved it by simply installing diffusers from PIP. Diffusers must be installed in virtual environment(venv) for a1111. To do that first activate venv. On windows machine go to a1111 folder, open command promp (type cmd in address bar of file explorer and hit enter), and run activate.bat script from venv\Scripts folder by typing command:
`venv\Scripts\activate.bat`
You should see "(venv)" before path in cmd window. Now you need to install the version 0.27.2 of diffusers that is required by IC-light project. The version of diffusers and other requirements can be found in https://github.com/lllyasviel/IC-Light/blob/main/requirements.txt . For installation simply type in cmd
`pip install diffusers==0.27.2`

Depending on you system you may need to install other requirements for IC-light. The procedure is the same "pip install" + copied line from requirements.txt
